pkgs/games/hexgui/default.nix

```diff
@@ -0,0 +1,34 @@
+{ ant
+, fetchFromGitHub
+, jdk
+, lib
+, makeWrapper
+, stdenv
+}:
+stdenv.mkDerivation {
+  pname = "hexgui";
+  version = "unstable-2022-5-30";
+
+  src = fetchFromGitHub {
+    owner = "selinger";
+    repo = "hexgui";
+    rev = "d94ce1d35a22dad28d3e7def4d28e6bebd54da9d";
+    hash = "sha256-1MroFH2JSEZHFigcsw1+xyHJWEnHTvHmRPVirUgwM6I=";
+  };
+
+  nativeBuildInputs = [ ant jdk makeWrapper ];
+  buildPhase = "ant";
+
+  installPhase = ''
+    mkdir $out;
+    cp -r bin lib $out
+    wrapProgram $out/bin/hexgui --prefix PATH : ${jdk}/bin
+  '';
+
+  meta = {
+    description = "HexGui";
+    longDescription = "GUI for the board game Hex (and Y)";
```

Comment:
```suggestion
    description = "GUI for the board game Hex (and Y)";
    homepage = "https://github.com/selinger/hexgui";
```

Reply:
So the fact that the actual name of the app is "HexGui" shouldn't be documented anywhere?

Reply:
I was going off of this https://github.com/NixOS/nixpkgs/blob/16f4e04658c2ab10114545af2f39db17d51bd1bd/pkgs/applications/version-management/gitlab/default.nix#L220-L221

Reply:
meta.description should be short but not that short.

> So the fact that the actual name of the app is "HexGui" shouldn't be documented anywhere?

Thats just a casing difference.